tslib/src/functions-throttled/ps-report-1-list-schools/index.ts

```diff
@@ -0,0 +1,25 @@
+import { AzureFunction, Context } from '@azure/functions'
+import { performance } from 'perf_hooks'
+import { SchoolListService } from './list-schools-service'
+
+const functionName = 'ps-report-1-list-schools'
```

Comment:
whats the purpose of the '1' in the name?

Reply:
According to the architecture, there will ultimately be 5 functions working on the PS report pipeline.  This naming scheme is 'ps-report-n-' as the prefix for these functions, where n indicates the pipeline stage.

I'm not committed to this naming scheme if you want to suggest something.